bin/typescript-fetch-petstore-all.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
```

Comment:
I would prefer to keep the {lang}-petstore-all.sh convention so as to make it consistent with other languages.